apps/web/ui/analytics/tag-selector.tsx

```diff
@@ -0,0 +1,71 @@
+import useTags from "@/lib/swr/use-tags";
+import { IconMenu, Popover, Tick, useRouterStuff } from "@dub/ui";
+import { ChevronDown, Tag } from "lucide-react";
+import Link from "next/link";
+import { useContext, useState } from "react";
+import { AnalyticsContext } from ".";
+import TagBadge from "../links/tag-badge";
+
+export default function TagSelector() {
```

Comment:
Pretty much just copied this from the interval selector in `apps/web/ui/analytics/toggle.tsx`